packages/create-project/commands/init/index.js

```diff
@@ -29,6 +29,20 @@ module.exports = class Project extends Generator {
     ).sort();
   }
 
+  _spawnSync(...args) {
+    const result = this.spawnCommandSync(...args);
+
+    if (result.error || result.status !== 0) {
+      const command = [args[0], ...args[1]].join(' ');
+
+      removeSync(this.options.directory);
+      this.log.error(result.error || new Error(`The command "${command}" exited unsuccessfully.`));
```

Comment:
Unless I use `--debug`, there is no indication of what actually went wrong, since the yarn stdout/stderr is never shown. I think we should also either:
1. Capture stdout/stderr and dump if there was an error
2. Tweak this error message to suggest using `--debug` to find out more